.github/workflows/CI.yml

```diff
@@ -146,24 +146,6 @@ jobs:
         run: dotnet fsi scripts/detectNotUsingGitPush1by1.fsx
       - name: Install prettier
         run: npm install prettier@2.8.3
-      - name: Change file permissions
-        # We need this step so we can change the files using `npx prettier --write` in the next step.
-        # Otherwise we get permission denied error in the CI.
-        run: sudo chmod 777 -R .
-      - name: Run "prettier" to check the style of our TypeScript and YML code
-        run: |
-          sudo npx prettier --quote-props=consistent --write './**/*.ts'
-          sudo npx prettier --quote-props=consistent --write './**/*.yml'
-          # Since we changed file modes in the previous step we need the following command to
-          # make git ignore mode changes in files and doesn't include them in the git diff command.
-          git config core.fileMode false
-          # Since after installing commitlint dependencies package.json file changes, we need to 
-          # run the following command to ignore package.json file
-          git restore package.json
-          git diff --exit-code
-      - name: fantomless
-        run: |
-          dotnet new tool-manifest
-          dotnet tool install fantomless-tool --version 4.7.997-prerelease
-          dotnet fantomless --recurse .
-          git diff --exit-code
+      - name: Check style of our F#, TypeScript and YML code
+        run: sudo dotnet fsi scripts/styleChecker.fsx
```

Comment:
@realmarv please make the `npm install prettier@2.8.3` also part of the script, but only run it if you detect there are .ts or .yml files in the repo (I know there are in this repo, but maybe in others where we will run this script, there won't be, so there's no need to install prettier in that case)

Reply:
Done